### PR TITLE
Prevent repeated crashes with socket activation

### PR DESCRIPTION
**Pull request by Scrumplex**
If an application shoves invalid input into the socket, the following input might be invalid as well.
systemd will, by default, send all pending input to the restarted process, in case the service crashed before.
We don't really want that, as it might cause wob to crash yet again. Therefore, systemd should just flush/discard all pending data and move on.

### CODE REVIEW

**Comment by francma** [comment]
Thanks, looks good.

One little note: please reword it to not say "crash", use "exit on invalid input" or something along these lines. Also the line with the comment in service file seems to be broken:

```
 If wob crashes from an input systemd should NOT shove it right back into it after it resta>
```

**Comment by Scrumplex** [comment]
Done! Also updated the commit message

**Comment by francma** [comment]
Thanks :+1:
